### PR TITLE
docs: E2E browser installation & setting up Node

### PR DESCRIPTION
**Pull request by supalarry**
## What does this PR do?

It documents:
1. How to resolve issue of not having test browser (Chromium) installed when running `yarn test-e2e`. [Commit](https://github.com/calcom/cal.com/pull/11551/commits/d331e64137f7750faabba9a23ad89ac072a5f312). 
2. How to use Node version required by Cal. Cal has `.nvmrc` file setup specifying the required Node version, and using nvm (Node Version Manager) it's possible to use Node at the version specified in `.nvmrc`. [Commit](https://github.com/calcom/cal.com/pull/11551/commits/4e67b13fdbf5fdebc99564aee8378ffbfdd64dd1).

### CODE REVIEW

**Review by Udit-takkar — DISMISSED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,7 +92,20 @@ To develop locally:
    - Use `openssl rand -base64 32` to generate a key and add it under `NEXTAUTH_SECRET` in the `.env` file.
    - Use `openssl rand -base64 24` to generate a key and add it under `CALENDSO_ENCRYPTION_KEY` in the `.env` file.
 
-6. Start developing and watch for code changes:
+6. Setup Node
+   If your Node version does not meet project's requirements as instructed by the docs, "nvm" (Node Version Manager) allows to use Node at the version required by the project:
```
Comment (keithwillcode) [inline CONTRIBUTING.md:96]: "the project's requirements"
"allows using Node"

File: README.md [hunk 2]
```diff
@@ -131,23 +131,36 @@ Here is what you need to be able to run Cal.com.
...
    ```
 
-1. Set up your `.env` file
+4. Set up your `.env` file
    - Duplicate `.env.example` to `.env`
    - Use `openssl rand -base64 32` to generate a key and add it under `NEXTAUTH_SECRET` in the `.env` file.
    - Use `openssl rand -base64 24` to generate a key and add it under `CALENDSO_ENCRYPTION_KEY` in the `.env` file.
 
+5. Setup Node
+   If your Node version does not meet project's requirements as instructed by the docs, "nvm" (Node Version Manager) allows to use Node at the version required by the project:
```
Comment (keithwillcode) [inline README.md:152]: "meet the project's"
"allows using Node"

**Review by keithwillcode — CHANGES_REQUESTED** [review]
Love the updates! Just had a couple tweaks to wording

File: CONTRIBUTING.md (hunk 1 above)
Comment (supalarry) [reply CONTRIBUTING.md]: 🫡 Yessir, resolved in [refactor: correct use of English](https://github.com/calcom/cal.com/pull/11551/commits/c5ae58af133f8c2b84d7f89563af47867dbee82f)

File: README.md (hunk 2 above)
Comment (supalarry) [reply README.md]: 🫡 Yessir, resolved in [refactor: correct use of English](https://github.com/calcom/cal.com/pull/11551/commits/c5ae58af133f8c2b84d7f89563af47867dbee82f)

**Review by keithwillcode — APPROVED** [review]

**Comment by supalarry** [comment]
> /tip 15 @supalarry

Thank you 🙏

**Pull request merged**